service/src/test/java/bio/terra/pipelines/dependencies/stairway/JobServiceMockTest.java

```diff
@@ -186,10 +191,11 @@ void retrieveJobResult_interrupted() throws InterruptedException {
   void retrieveJob_stairwayException() throws InterruptedException {
```

Comment:
I just noticed these test method names being both camel and snake case.  We should take a pass at these and make them consistent (camel case preferred)

Reply:
this has been bothering me for a while. also some test names start with "test" and others don't. I'm refactoring them to remove "test" from the names as well as remove snake case.